w04h02/test/pgdp/megamerge/Task1Tests.java

```diff
@@ -0,0 +1,69 @@
+package pgdp.megamerge;
```

Comment:
Hey! Thanks for your PR! Is it correct, that you are adding tests for W04H02?

Reply:
That commit was probably added by a rebase on the main branch.

Reply:
Can we remove them from the PR? You can just delete them on the branch `add-tests-for-w05h03` of the fork